### PR TITLE
Switch to stroke symbol type for line marks -- Fix #4674

### LINKED ISSUE #4674: Use symbolType = stroke for line marks
From Slack

> if generating a legend for a line mark, consider using the new `"symbolType": "stroke"` option instead of `"circle"`! There’s an example usage here: https://vega.github.io/vega/examples/edge-bundling/

### PR DESCRIPTION
**Pull request by kanitw**

### CODE REVIEW

**Comment by kanitw** [comment]
Question: should line overlay use a symbol or stroke? 

image

**Comment by domoritz** [comment]
Yes, symbol! 

The lines are also pretty thin. I wonder whether we want to/can make it thicker.

**Comment by domoritz** [comment]
Yes, symbol! 

The lines are also pretty thin. I wonder whether we want to/can make it thicker.

File: examples/compiled/facet_independent_scale_layer_broken.vg.json [hunk 1]
```diff
@@ -246,6 +246,7 @@
     {
```
Comment (domoritz) [inline examples/compiled/facet_independent_scale_layer_broken.vg.json]: How is this one broken?

Comment (kanitw) [reply examples/compiled/facet_independent_scale_layer_broken.vg.json]: IDK lol

**Comment by domoritz** [comment]
Build is broken.

**Comment by kanitw** [comment]
Hmm, still incorrect.

**Comment by kanitw** [comment]
should be good now, I hope.

**Pull request merged**